### PR TITLE
Extract the content of the case 'Int32' (Flow, TypeScript) into a single emitInt32 function in the parsers-primitives.js file.

### PR DESCRIPTION
**Pull request by Naturalclar**
## Summary

Part of #34872

Refactors emitInt32 in codegen parser module into a common parsers-primitive.js

## Changelog

[Internal] [Changed] - Extract the content of the case 'Int32' (Flow, TypeScript) into a single emitInt32 function

## Test Plan

Ran `yarn jest react-native-codegen` 

Screen Shot 2022-10-09 at 15 45 21

### CODE REVIEW

**Comment by Naturalclar** [comment]
@cipolleschi 
Thanks for the review! I've added a unit test for emitInt32.

File: packages/react-native-codegen/src/parsers/__tests__/parser-primitive-test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
```
Comment (rshest) [inline packages/react-native-codegen/src/parsers/__tests__/parser-primitive-test.js:1]: Thanks for adding the test!
Could you please name this `parser-primitives-test.js`, for consistency?

**Review by rshest — CHANGES_REQUESTED** [review]
Please rename `parser-primitive-test.js` -> `parsers-primitives-test.js`. Thanks!

File: packages/react-native-codegen/src/parsers/__tests__/parser-primitive-test.js (hunk 1 above)
Comment (Naturalclar) [reply packages/react-native-codegen/src/parsers/__tests__/parser-primitive-test.js]: @rshest thanks for the review!
I realized that the original file was called `parsers-primitives.js`, both word containing `s` 
So I renamed the test to also follow the same `parsers-primitives-test.js`

**Pull request closed without merging**